6MI0800226/week1.cpp

```diff
@@ -0,0 +1,114 @@
+#include <iostream>
+#pragma warning(disable: 4996)
+struct transition
+{};
+
+
+//zad 2
+struct pair
+{
+    int firstInt;
+    int secondInt;  
+};
+void inputIntsOfPair(pair& pair1)
+{
+    std::cout << "First integer: ";
+    std::cin >> pair1.firstInt;
+    std::cout << "Second integer: ";
+    std::cin >> pair1.secondInt;
+}
+bool isBigger(const pair& firstPair, const pair& secondPair)
+{
+    return firstPair.firstInt > secondPair.firstInt ||
+        (firstPair.firstInt == secondPair.firstInt && firstPair.secondInt > secondPair.secondInt);
+}
+pair biggestPair(const pair* pairs, size_t size)
```

Comment:
Overall leaving an empty line here and there to be used to visually separate different functions etc. is not a bad idea